### DISCUSSION TITLE
May can meson build synopsys metaware arc compilier?

### DISCUSSION THREAD

**Discussion by Nell69Rock** [mesonbuild/meson#10205] [Q&A] [answered]
watching this site.

https://mesonbuild.com/Reference-tables.html

ccac compiler is not supported yet . only support arc cpu family!

how can i use ccac(metaware compiler) ?

**Comment by eli-schwartz** [comment, marked as answer]
Someone with access to this compiler would need to add a class for it in `mesonbuild/compilers/c.py` (there's a "clike" mixin that probably does much of the work) and register it in `mesonbuild/compilers/detect.py`. If you are interested, we could provide further guidance.

**Reply by Nell69Rock** [reply]
ok, I'll try it first and reply to you. thx answer for me

**Reply by Nell69Rock** [reply]
can i use your provided futher guidance?
